source/documentation/rules-b.html.md.erb

```diff
@@ -0,0 +1,668 @@
+
+##B1203
+
+<b>Functional Description</b>
+
+N/A
+
+<b>Technical Description</b>
+
+IF &lt;Decisive Date&gt; is LESS than or EQUAL to &lt;TPendDate&gt; <br/>THEN<br/>/CD018C/Consignment/HouseConsignment/ConsignmentItem/Packaging/shippingMarks<br/><br/>format shall be set to an..42<br/>
+
+##B1206
+
+<b>Functional Description</b>
+
+N/A
+
+<b>Technical Description</b>
+
+IF &lt;Decisive Date&gt; is LESS than or EQUAL to &lt;TPendDate&gt; <br/>THEN<br/>/CD018C/Consignment/grossMass AND<br/>/CD018C/Consignment/HouseConsignment/ConsignmentItem/Commodity/GoodsMeasure/grossMass AND<br/>/CD018C/Consignment/HouseConsignment/ConsignmentItem/Commodity/GoodsMeasure/netMass <br/><br/>format shall be set to n..11,3<br/><br/>
+
+##B1401
+
+<b>Functional Description</b>
+
+N/A
+
+<b>Technical Description</b>
+
+IF &lt;Decisive Date&gt; is LESS than or EQUAL to &lt;TPendDate&gt;<br/>THEN<br/>for each Goods Item, the cumulated number of all instances of<br/>/CD018C/Consignment/HouseConsignment/ConsignmentItem/SupportingDocument AND<br/>/CD018C/Consignment/HouseConsignment/ConsignmentItem/TransportDocument AND<br/>/CD018C/Consignment/HouseConsignment/ConsignmentItem/AdditionalReference<br/>shall be LESS OR EQUAL to 99x
+
+##B1804
+
+<b>Functional Description</b>
+
+N/A
+
+<b>Technical Description</b>
+
+IF &lt;Decisive Date&gt; is LESS than or EQUAL to &lt;TPendDate&gt;<br/>THEN C0710 attached to /*/Consignment/LocationOfGoods<br/>shall be disabled AND<br/>/*/Consignment/LocationOfGoods = "O"<br/>
+
+##B1805
+
+<b>Functional Description</b>
+
+N/A
+
+<b>Technical Description</b>
+
+IF &lt;Decisive Date&gt; is LESS than or EQUAL to &lt;TPendDate&gt;<br/>THEN C0837 will be disabled AND<br/>  IF /*/ TransitOperation/reducedDatasetIndicator is EQUAL TO ‘1’<br/>     THEN<br/>     /*/Consignment/HouseConsignment/ConsignmentItem/Commodity/GoodsMeasure/netMass = "N"<br/>  ELSE         <br/>  /*/Consignment/HouseConsignment/ConsignmentItem/Commodity/GoodsMeasure/netMass = "O"
+
+##B1806
+
+<b>Functional Description</b>
+
+N/A
+
+<b>Technical Description</b>
+
+IF &lt;Decisive Date&gt; is LESS than or EQUAL to &lt;TPendDate&gt;<br/>THEN C0806 will be disabled AND<br/>IF /*/Consignment/modeOfTransportAtTheBorder is PRESENT<br/>THEN<br/>   IF /*/Consignment/modeOfTransportAtTheBorder is EQUAL to '5' or '2'<br/>   THEN /*/Consignment/ActiveBorderTransportMeans = “O”<br/>   ELSE /*/Consignment/ActiveBorderTransportMeans = “R”<br/>ELSE<br/>/*/Consignment/ActiveBorderTransportMeans = “O”
+
+##B1811
+
+<b>Functional Description</b>
+
+N/A
+
+<b>Technical Description</b>
+
+IF &lt;Decisive Date&gt; is LESS than or EQUAL to &lt;TPendDate&gt;<br/>THEN R0076 attached to <br/>/*/Consignment/ActiveBorderTransportMeans/identificationNumber<br/>will be disabled
+
+##B1813
+
+<b>Functional Description</b>
+
+N/A
+
+<b>Technical Description</b>
+
+IF &lt;Decisive Date&gt; is LESS than or EQUAL to &lt;TPendDate&gt; <br/>THEN R0003 and R0006 shall be disabled. 
```

Comment:
Consistency issue - This sentence has a full stop but most other sentences here don't see to have full stops. We need to be consistent.